src/py/flwr/client/grpc_client/connection.py

```diff
@@ -118,7 +129,48 @@ def grpc_connection(
     server_message_iterator: Iterator[ServerMessage] = stub.Join(iter(queue.get, None))
 
     def receive() -> TaskIns:
-        server_message = next(server_message_iterator)
+        # Receive ServerMessage proto
+        proto = next(server_message_iterator)
+
+        # ServerMessage proto --> *Ins --> RecordSet
+        field = proto.WhichOneof("msg")
+        task_type = ""
+        if field == "get_properties_ins":
+            recordset = compat.getpropertiesins_to_recordset(
+                serde.get_properties_ins_from_proto(proto.get_properties_ins)
+            )
```

Comment:
Not important for this PR, but we should support `keep_input` in all of our conversion functions:
```suggestion
            recordset = compat.getpropertiesins_to_recordset(
                serde.get_properties_ins_from_proto(proto.get_properties_ins), False
            )
```